lib/ui/alert_detail.dart

```diff
@@ -339,7 +339,9 @@ class _AlertDetailPageState extends State<AlertDetailPage> {
                     ].expand(
                           (widget) => [
                         widget,
-                        const SizedBox(
+                        (widget.runtimeType == Offstage && (widget as Offstage).offstage)
```

Comment:
Just out of curiosity, is the condition of `widget.runtimeType == Offstage` good enough? Not two sure about the difference between the first condition and the second one :)

Reply:
> Just out of curiosity, is the condition of `widget.runtimeType == Offstage` good enough to check whether it's offstage? Not too sure about the difference between the first condition and the second one :)

From reading the doc i think the first == Offstage check if for the type, second  .offstage is checking its state

Reply:
Yeah the naming convention is bit weird, I'll never name a field the same as the class.